### DISCUSSION TITLE
bt2 works with garnet & lispworks!

### DISCUSSION THREAD

**Discussion by albuspiroglu** [sionescu/bordeaux-threads#100] [Show and tell]
I wanted to share my excitement when today, I finally made garnet run on lispworks. I used bt2, as I couldn't succeed with bt (it was hanging during gem:init, which I haven't identified the root cause for.

**Comment by albuspiroglu** [comment]
Here is the branch that works: https://github.com/albuspiroglu/cl-garnet/tree/albus-bordeax-threads-2

**Comment by sionescu** [comment]
Very nice !
